examples/stress_tests/many_cubes.rs

```diff
@@ -262,6 +278,108 @@ fn init_materials(
     materials
 }
 
+fn init_meshes(args: &Args, assets: &mut Assets<Mesh>) -> Vec<(Handle<Mesh>, Transform)> {
+    let capacity = args.mesh_count.max(1);
+
+    let mut meshes = Vec::with_capacity(capacity);
+
+    let mut radius_rng = StdRng::seed_from_u64(42);
+    let mut variant = 0;
+    meshes.extend(
+        std::iter::repeat_with(|| {
+            let radius = radius_rng.gen_range(0.25f32..=0.75f32);
+            let (handle, transform) = match variant % 15 {
+                0 => (
```

Comment:
Wouldn't you want to only vary the uniqueness of the mesh and not the overall triangle count? Couldn't you use a bunch of separate instances of same-size cubes?

Reply:
I could, but I'd like to use a variety of meshes as well. As they are randomly distributed, the per-frame draws should be fairly consistent, if that's what you're concerned about?

Reply:
The concern is mainly "changing more than one variable in an experiment."
  
This switch adds a bunch of triangles to the scene, in addition to making the mesh assets unique. So it seems like like comparisons between runs where this switch is enabled and not enabled would not be useful.

I had assumed based on:
> This breaks batches based on different meshes.

That just making the assets unique was the main goal.

If that's not the case or not a worry, then I guess I'm not bothered.

Reply:
My goal was not really to compare single mesh vs many meshes, rather just to create a scene with a bunch of different meshes. The benchmarking is done by using the same configuration of the stress test with two versions of the code.